Gems/PhysX/Code/Include/PhysX/Joint/Configuration/PhysXJointConfiguration.h

```diff
@@ -84,6 +84,22 @@ namespace PhysX
         float m_tolerance = 0.1f; ///< Distance from the joint at which limits becomes enforced. Used if limit is hard.
     };
 
+    struct JointMotorProperties
+    {
+        AZ_CLASS_ALLOCATOR(JointMotorProperties, AZ::SystemAllocator, 0);
+        AZ_TYPE_INFO(JointMotorProperties, "{9CF35393-82CD-4726-B387-96F6381046B3}");
+        static void Reflect(AZ::ReflectContext* context);
+
+        JointMotorProperties() = default;
+        JointMotorProperties(
+            bool enabled, float gear_ratio, float force_limit);
```

Comment:
[standard] name convention for parameters:
gear_ratio -> gearRatio
force_limit -> forceLimit